gcloud/monitoring/test_client.py

```diff
@@ -0,0 +1,325 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest2
+
+PROJECT = 'my-project'
+
+
+class TestClient(unittest2.TestCase):
+
+    def _getTargetClass(self):
+        from gcloud.monitoring.client import Client
+        return Client
+
+    def _makeOne(self, *args, **kwargs):
+        return self._getTargetClass()(*args, **kwargs)
+
+    def test_query(self):
+        import datetime
+
+        START_TIME = datetime.datetime(2016, 4, 6, 22, 5, 0)
+        END_TIME = datetime.datetime(2016, 4, 6, 22, 10, 0)
+        MINUTES = 5
+
+        METRIC_TYPE = 'compute.googleapis.com/instance/cpu/utilization'
+        METRIC_LABELS = {'instance_name': 'instance-1'}
+        METRIC_LABELS2 = {'instance_name': 'instance-2'}
+
+        RESOURCE_TYPE = 'gce_instance'
+        RESOURCE_LABELS = {
+            'project_id': 'my-project',
+            'zone': 'us-east1-a',
+            'instance_id': '1234567890123456789',
+        }
+        RESOURCE_LABELS2 = {
+            'project_id': 'my-project',
+            'zone': 'us-east1-b',
+            'instance_id': '9876543210987654321',
+        }
+
+        METRIC_KIND = 'GAUGE'
+        VALUE_TYPE = 'DOUBLE'
+
+        TS1 = '2016-04-06T22:05:00.042Z'
+        TS2 = '2016-04-06T22:05:01.042Z'
+        TS3 = '2016-04-06T22:05:02.042Z'
+
+        VAL1 = 0.1
+        VAL2 = 0.2
+
+        def P(timestamp, value):
```

Comment:
Suggest renaming `P` as `make_point` or similar, though this is a `test_*.py` module so it's kind of the wild west, so you can leave it if you like.